### DISCUSSION TITLE
Api to get the fileClass when using tags

### DISCUSSION THREAD

**Discussion by LeonardoGentile** [mdelobelle/metadatamenu#635] [Q&A]
When simply assigning a class using the `fileClass` property it is easy to retrieve all the instances in dataview filtering by this field. 
But if I chose the option to map with a tag and especially if the tag name is not the same as the class name how can I tell which class a particular note belongs to (if any).
On a broader way how to retrieve all the notes of a particular class when the class is assigned via tags?

**Comment by msageryd** [comment]
+1
I needed this in a custom plugin, but I gave up. The class is assigned, but I can't find any evidence of this in the actual md file. One solution might be for medatadaMenu to automatically add the assigned class in the class property in frontmatter, even if the origin of the assignment is tag or bookmarkGroup.
